browser/src/Services/Explorer/ExplorerFileSystem.ts

```diff
@@ -24,12 +32,21 @@ export class FileSystem implements IFileSystem {
         stat(path: string): Promise<fs.Stats>
         exists(path: string): Promise<boolean>
     }
+    private _backupDirectory = path.join(tempdir(), "oni_backup")
+
+    public get backupDir(): string {
+        return this._backupDirectory
+    }
 
-    constructor(nfs: typeof fs) {
+    constructor(nfs: typeof fs, _promisify: typeof promisify) {
```

Comment:
@DeltaEvo `jest` isn't resolving `promisify` since its used inside of the class, so I'm passing it in explicitly so I can use it in the test.

My most recent changes btw are a work in progress as I need to resolve changes to the filesystem.